crates/core/tedge_actors/Cargo.toml

```diff
@@ -10,6 +10,7 @@ homepage = { workspace = true }
 repository = { workspace = true }
 
 [dependencies]
+anyhow = "1.0.69"
```

Comment:
That looks wrong. We've been trying to remove this dependency so far.

Reply:
This was removed, forgot to push the changes.